contrib/build-venv.sh

```diff
@@ -3,7 +3,7 @@
 VENV=$(dirname $0)/..
 BUILD=${VENV}/build
 DIST=${VENV}/dist
-EXTRA_ARGS="-Dlibxmlb:gtkdoc=false -Dsystemd=disabled -Dlaunchd=disabled"
+EXTRA_ARGS="-Dlibxmlb:gtkdoc=false -Dsystemd=disabled -Dlaunchd=disabled -Dbinder=enabled"
```

Comment:
Is this really needed?  It's a feature, so I think that meson will generally default to enabled if all dependencies are met won't it?